packages/nimble-components/src/table/specs/README.md

```diff
@@ -133,13 +133,25 @@ The various APIs/features of the `nimble-table` will be split up amongst several
     -   Define how the action menu gets associated with a particular column
     -   Define the table-level(column-level?) API(s) for applying an action menu to the table (slot, properties, etc...)
 
+_Attributes_
+
+-   `id-field-name` - An optional string attribute that specifies the field name within a row's record to use as a row's ID. If the attribute is not specified, a default ID will be generated. If the attribute is invalid, no rows in the table will be rendered, and the table will enter an invalid state according to the `validity` property and `checkValidity()` function. The attribute is invalid in the following conditions:
```

Comment:
Good thought that since the generated id does expose itself in the public api (events, selected ids, etc) we should have control over it. Maybe we should capture some notes in open issues about that?

Reply:
I've added a note in the `Open Issues` section.